docs/transparent-validation/pikvm/usage-validation.md

```diff
@@ -0,0 +1,151 @@
+# PiKVM usage
+
+## First use
+
+> The following procedure should be performed not only before the first use of
+the device, but also as the presale validation procedure!
+
+### Hardware requirements
+
+* Raspberry Pi Zero2W,
+* microSD card 16 GB,
+* USB-A 5 V 3.1 A charger (female socket),
+* HDMI to CSI-2 bridge,
+* Raspberry Pi Zero Camera Cable,
+* HDMI-HDMI cable,
+* USB A - micro USB cable (male, male),
+* Y-splitter cable.
+* (Optional) UART -> USB converter
+
+### Building the OS
+
+To prevent any mix-up in libraries and dependencies process will include Docker.
+
+1. Install and configure Docker:
+
+    ```bash
+    sudo apt-get install git make curl binutils -y
+    curl -fsSL https://get.docker.com -o get-docker.sh
+    sudo sh get-docker.sh
+    sudo usermod -aG docker $USER
+    ```
+
+    - Restart PC or relogin to apply changes.
+
+1. Get OS repository:
+
+    ```bash
+    git clone --depth=1 https://github.com/pikvm/os
+    cd os
+    ```
+
+1. Create `config.mk` file as follows:
+
+    ```bash
+    BOARD = zero2w
+
+    PLATFORM = v2-hdmi
+
+    HOSTNAME = pikvm
+
+    LOCALE = en_US
+
+    TIMEZONE = Europe/Warsaw
+
+    ROOT_PASSWD = root
+
+    WEBUI_ADMIN_PASSWD = admin
+
+    IPMI_ADMIN_PASSWD = admin
+
+    CARD = <SD_card_location>
+    ```
+
+1. Build the OS:
+
+    ```bash
+    make os
+    ```
+
+    - If any error occurs, see
+    [original documentation](https://github.com/pikvm/pikvm/blob/master/docs/building_os.md).
+
+1. Create the Image:
+
+    ```bash
+    make image
+    ```
+
+1. Install the image on the SD card using eg.
```

Comment:
In this section, we should use our standard description of the flashing SDcard procedure. Refer: https://docs.dasharo.com/transparent-validation/rte/v1.1.0/quick-start-guide/#preparation-of-rte

Reply:
done